### ISSUE TITLE
aiops

### ISSUE THREAD

**Issue by heidsoft** [heidsoft/cloud-bigdata-book#98]
https://machinelearningmastery.com/develop-first-xgboost-model-python-scikit-learn/
https://e.huawei.com/en/publications/global/ict_insights/201810161444/customers-on-digital-transformation/201810161635

https://zhuanlan.zhihu.com/p/90790966

https://www.logicmonitor.com/lm-difference

https://xgboost.readthedocs.io/en/latest/python/python_intro.html#python-data-interface

https://www.jianshu.com/p/31e20f00c26f?spm=5176.12282029.0.0.36241491UUhnZE

https://zhuanlan.zhihu.com/p/67832773

https://github.com/dmlc/xgboost/tree/master/demo/data

https://www.cnblogs.com/mantch/p/11164221.html

https://blog.csdn.net/u011630575/article/details/79418138

https://www.cnblogs.com/pinard/p/11114748.html  XGBoost类库使用小结

https://www.ctolib.com/topics-124853.html

https://aihub.cloud.google.com/p/products%2F3e82a569-8cdb-40ea-b7db-4fd5edcc0c2a  Python机器学习实战之手撕XGBoost

http://www.huaxiaozhuan.com/%E5%B7%A5%E5%85%B7/xgboost/chapters/xgboost_usage.html  Xgboost使用

https://www.datacamp.com/community/tutorials/xgboost-in-python

数据格式
https://qastack.cn/stats/61328/libsvm-data-format

https://blog.csdn.net/yangshaojun1992/article/details/87861767

https://blog.csdn.net/kobesdu/article/details/8944851  生成libSVM的数据格式及使用方法总结

libsvm文件格式

libsvm数据格式 libsvm使用的训练数据和检验数据文件格式如下：

[label] [index1]:[value1] [index2]:[value2] …

[label] [index1]:[value1] [index2]:[value2] … label 目标值，就是说class（属于哪一类），就是你要分类的种类，通常是一些整数。

index 是有顺序的索引，通常是连续的整数。就是指特征编号，必须按照升序排列

value 就是特征值，用来train的数据，通常是一堆实数组成。

csv to libsvm
https://zhuanlan.zhihu.com/p/96042647

训练
https://zhuanlan.zhihu.com/p/75133195 模型训练与可视化

https://blog.csdn.net/u011630575/article/details/79418138

**Comment by heidsoft** [comment]
https://docs.seldon.io/projects/seldon-core/en/v1.1.0/index.html
https://www.kubeflow.org/docs/components/serving/bentoml/
https://www.kubeflow.org/docs/about/kubeflow/
https://docs.bentoml.org/en/latest/frameworks.html#xgboost
https://opendatahub.io/docs/kubeflow/installation.html
https://www.kubeflow.org/docs/about/kubeflow/

**Comment by heidsoft** [comment]
https://mlflow.org/docs/latest/tracking.html
https://zhuanlan.zhihu.com/p/57639703
https://cloud.tencent.com/developer/article/1692441
https://prometheus.io/docs/concepts/data_model/#samples
https://en.wikipedia.org/wiki/Time_series
https://prometheus.io/docs/concepts/jobs_instances/
https://github.com/AICoE/prometheus-data-science
https://github.com/AICoE
https://github.com/AICoE/prometheus-data-science/blob/master/metadata_analysis/graph_metadata.py
https://github.com/durandom/DataScience-on-Prometheus-Metrics#forcasting
https://www.tensorflow.org/io/tutorials/prometheus
https://next.redhat.com/2019/11/18/prometheus-anomaly-detection/
https://github.com/mlflow/mlflow/
https://thoth-station.ninja/
https://next.redhat.com/category/ai-machine-learning/
https://github.com/AICoE/prometheus-anomaly-detector
https://mlflow.org/docs/latest/quickstart.html#installing-mlflow
